task8/airline/models/airlines.py

```diff
@@ -0,0 +1,64 @@
+import collections
+
+from airline import exceptions
+
+
+class Airline(collections.deque):
+    def __init__(self, name, initial_aircrafts):
+        self.name = name
+        self.types = initial_aircrafts.keys()
+        super().__init__(self._initial_values(initial_aircrafts))
+
+    def _initial_values(self, initial_aircrafts):
+        aircrafts = []
+        for class_obj, count in initial_aircrafts.items():
+            for _ in range(count):
+                aircrafts.append(
+                    class_obj(id=len(aircrafts) + 1, name=f'Default {class_obj.__name__}', location=self.name))
+        return aircrafts
+
+    def append(self, *args):
+        for aircraft in args:
+            if isinstance(aircraft, tuple(self.types)):
+                aircraft.id = len(self) + 1
+                self.append(aircraft)
+            else:
+                raise exceptions.ObjectIsNotAircraftError
+
+    def total_capacity(self):
+        return sum(aircraft.capacity for aircraft in self)
+
+    def sort_by_range(self):
+        return sorted(self, key=lambda aircraft: aircraft.max_range)
+
+    def get_by_id(self, id):
+        if id <= len(self) - 1:
```

Comment:
Are you sure that it safe?
What happen If we delete aircraft by index and after add new one?